docs/dev.rst

```diff
@@ -156,37 +192,31 @@ You are then free to bind-mount any directory to any container. For instance, to
         volumes:
           - /path/to/edx-platform:/openedx/edx-platform
 
-This override file will be loaded when running any ``tutor dev ..`` command. The edx-platform repo mounted at the specified path will be automatically mounted inside all LMS and CMS containers. With this file, you should no longer specify the ``-v/--volume`` option from the command line with the ``run`` or ``runserver`` commands.
+This override file will be loaded when running any ``tutor dev ..`` command. The edx-platform repo mounted at the specified path will be automatically mounted inside all LMS and CMS containers. With this file, you should no longer specify the ``-m/--mount`` option from the command line.
 
 .. note::
-    The ``tutor local`` commands load the ``docker-compose.override.yml`` file from the ``$(tutor config printroot)/env/local/docker-compose.override.yml`` directory.
+    The ``tutor local`` commands load the ``docker-compose.override.yml`` file from the ``$(tutor config printroot)/env/local/docker-compose.override.yml`` directory. One-time jobs from initialisation commands load the ``local/docker-compose.jobs.override.yml`` and ``dev/docker-compose.jobs.override.yml``.
 
 Common tasks
 ------------
 
+.. _edx_platform_dev_env:
+
 Setting up a development environment for edx-platform
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Following the instructions :ref:`above <bind_mounts>` on how to bind-mount directories from the host above, you may mount your own `edx-platform <https://github.com/openedx/edx-platform/>`__ fork in your containers by running either::
-
-    # Mount from the volumes/ directory
-    tutor dev bindmount lms /openedx/edx-platform
-    tutor dev runserver --volume=/openedx/edx-platform lms
-
-    # Mount from an arbitrary directory
-    tutor dev runserver --volume=/path/to/edx-platform:/openedx/edx-platform lms
+Following the instructions :ref:`above <bind_mounts>` on how to bind-mount directories from the host above, you may mount your own `edx-platform <https://github.com/openedx/edx-platform/>`__ fork in your containers by running::
 
-    # Add your own volumes to $(tutor config printroot)/env/dev/docker-compose.override.yml
-    tutor dev runserver lms
+    tutor dev runserver --mount=/path/to/edx-platform lms
```

Comment:
Since the linked bind-mount instructions use `start`, let's use it here too.
```suggestion
    tutor dev start -d --mount=/path/to/edx-platform lms
```